fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/ApplicationAmbassador.java

```diff
@@ -608,7 +610,7 @@ private void process(final TrafficLightUpdates trafficLightUpdates) {
                 addEvent(event);
             }
         }
-
+        SimulationKernel.SimulationKernel.getCentralPerceptionComponentComponent().updateTrafficLights(trafficLightUpdates);
```

Comment:
the name of the method getCentralPerceptionComponentComponent contains "component" twice